utils_nlp/eval/rouge_ext.py

```diff
@@ -0,0 +1,978 @@
+# This script is adopted from https://github.com/Diego999/py-rouge/blob/master/rouge/rouge.py
```

Comment:
copyright

Reply:
can all the python files related to rouge move to a folder,  eval/rouge?

Reply:
According to CELA, we don't need to add the MSFT copyright if we don't make significant changes to a piece of code, which is the case here.
I created a rouge folder and moved rouge related scripts there.